canonical/ubuntu.com
# WD-3419 Fix links across the site

## Done

- Update broken links on production

## QA

- Check the new redirect works
- Check all links [reported](https://github.com/canonical/ubuntu.com/actions/runs/4762485451/jobs/8464764955) in this check have been fixed
